config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
+  use_doorkeeper do
+    skip_controllers :authorizations, :authorized_applications, :tokens, :token_info
+  end
+
   devise_for :users
   
   namespace :api do
     namespace :v1 do
       resources :keywords, only: :index
+      get 'private/sample', to: 'private_items#index'
```

Comment:
Is there a reason why we use a custom route? 👀 
While it is not forbidden, we generally try to stay within the basic resources routes, unless there is a specific need that cannot use them (e.g. a download file action that is not specific to a single `id` – so cannot use the `show` default route).

Note that this is not a big issue, but more a small practice to stay as close as possible to the Rails way ;-)